pkg/libartifact/store/store.go

```diff
@@ -160,7 +161,9 @@ func (as ArtifactStore) Push(ctx context.Context, src, dest string, opts libimag
 
 // Add takes one or more local files and adds them to the local artifact store.  The empty
 // string input is for possible custom artifact types.
-func (as ArtifactStore) Add(ctx context.Context, dest string, paths []string, _ string) (*digest.Digest, error) {
+func (as ArtifactStore) Add(ctx context.Context, dest string, paths []string, options *libartTypes.AddOptions) (*digest.Digest, error) {
+	annots := make(map[string]string)
+	maps.Copy(annots, options.Annotations)
```

Comment:
maps.Clone() is much better to read an understand here.

`annots := maps.Clone(options.Annotations)`